include/swift/AST/ASTBridging.h

```diff
@@ -1037,6 +1038,26 @@ BridgedVarDecl_asAbstractStorageDecl(BridgedVarDecl decl);
 // MARK: Exprs
 //===----------------------------------------------------------------------===//
 
+struct BridgedCallArgument {
```

Comment:
`BridgedArgument` is already used in `SILBridging`.

Reply:
There are a bunch of types like that in SILBridging unfortunately, we probably ought to rename them to match their C++ counterpart rather than their Swift counterpart